controllers/dependencies.go

```diff
@@ -52,16 +56,15 @@ func (r *OperatorPipelineReconciler) reconcilePipelineDependencies(meta metav1.O
 
 				// apply pipeline yaml manifests
 				if path == PIPELINE_MANIFESTS_PATH {
-					if errors := r.applyManifests(filePath, meta.Namespace, &pipeline); errors != nil {
-						log.Error(errors, "Couldn't apply pipeline manifest")
-						return errors
+					if errs := r.applyPipelineManifests(filePath, meta, &pipeline); errs != nil {
+						log.Error(errs, "Couldn't apply pipeline manifest")
+						return errs
 					}
-
 					// or apply tasks manifests
 				} else {
-					if errors := r.applyManifests(filePath, meta.Namespace, &task); errors != nil {
-						log.Error(errors, "Couldn't apply task manifest")
-						return errors
+					if errs := r.applyTaskManifests(filePath, meta, &task); errs != nil {
+						log.Error(errs, "Couldn't apply task manifest")
```

Comment:
this could be removed. applyManifests prints enough logging messages.

Reply:
Done.